src/test/java/bio/terra/service/filedata/azure/AzureSynapsePdaoConnectedTest.java

```diff
@@ -113,6 +130,26 @@ public void setup() throws Exception {
             .name(STORAGE_ACCOUNT_NAME)
             .applicationResource(applicationResource)
             .metadataContainer("metadata");
+
+    client =
+        azureResourceConfiguration.getClient(
+            azureResourceConfiguration.getCredentials().getHomeTenantId(),
+            billingProfile.getSubscriptionId());
+
+    StorageAccount storageAccount =
+        client
+            .storageAccounts()
+            .define("ct" + Instant.now().toEpochMilli())
```

Comment:
Why a time-related identifier rather than a random UUID?

Reply:
I copied this from another test...a UUID works too!

Reply:
Ah this needs to be an alphanumeric string between 3-24 characters so I just changed it back

Reply:
oh yeah. Ack!